# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/no-kill.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./MCONFIG.orig	Wed Apr 10 16:01:13 2002
+++ ./MCONFIG	Thu Apr 25 12:44:06 2002
@@ -73,7 +73,7 @@
 # If HAVE_KILL is set to "yes", then kill will not be built or
 # installed from the misc-utils subdirectory.
 # (There is also a kill in the procps package.)
-HAVE_KILL=no
+HAVE_KILL=yes
 
 # If ALLOW_VCS_USE is set to "yes", then login will chown /dev/vcsN
 # to the current user, allowing her to make a screendump and do other
